tests/templates/cert-manager-aio-test.yaml

```diff
@@ -42,6 +42,8 @@ metadata:
     app: microbot
   name: microbot
 spec:
+  externalTrafficPolicy: Cluster
+  type: LoadBalancer
```

Comment:
This says we will need a loadbalancer to get this service up. What is the error we get if we do not do this? Does the test that check the cert-manager install metallb?

Reply:
We were not able to finish the challenge (`Waiting for http-01 challenge propagation`) and it was a continuous error. This was probably caused due to non-resolution of the host `microbot.100.100.100.100.nip.io`.